chandra_aca/star_probs.py

```diff
@@ -259,74 +239,73 @@ def acq_success_prob(date=None, t_ccd=-19.0, mag=10.0, color=0.6, spoiler=False,
         warm_fracs.append(warm_frac)
     warm_frac = np.array(warm_fracs).reshape(dates.shape)
 
-    probs = model_acq_success_prob(mags, warm_fracs, colors)
+    probs = model_acq_success_prob(mags, warm_fracs, colors, halfwidths)
 
     p_0p7color = .4294  # probability multiplier for a B-V = 0.700 star (REF?)
     p_spoiler = .9241  # probability multiplier for a search-spoiled star (REF?)
 
     # If the star is brighter than 8.5 or has a calculated probability
     # higher than the max_star_prob, clip it at that value
-    max_star_prob = .985
-    probs[mags < 8.5] = max_star_prob
+    probs[mags < 8.5] = MAX_ACQ_PROB
```

Comment:
Oops, I see that clipping is down below anyway, so is this 8.5 limit just because nothing reasonable is fit brighter than 8.5?